build.py

```diff
@@ -95,7 +95,7 @@ def cmd(cmd, quiet=False, shell=False):
 kernel_release = cmd('uname -r', quiet=True).strip()
 
 DPDK_DIR = '%s/%s' % (DEPS_DIR, DPDK_VER)
-DPDK_CFLAGS = '"-g -w -fPIC"'
+DPDK_CFLAGS = '"-g -w"'
```

Comment:
We might (on some systems that we don't really care about...?) need an explicit `no-pic` and/or `no-pie` option (eg `-fno-PIE`). It looks like there was a big mess of contradictory options introduced in various distributions back in 2016 or so. Should we have a build.py flag for this?  That is, rather than hardcoding `-fPIC` or absence of `-fPIC` we could have an option to put something in.

We could perhaps even try to figure out the right flags, if any, automatically...  If this works for Ubuntu 18 though, that's what we really care about and should be the default.

Reply:
"Ubuntu18 with gcc+8" doesn't support "-fPIC", and doesn't necessarily require explicit `no-pie` option. For the older version of g++, yeap, we explicitly disable the `no-pie` sounds better option to overwrite the default as you said.